Test/Import-WhiskeyPowerShellModule.Tests.ps1

```diff
@@ -0,0 +1,137 @@
+Set-StrictMode -Version 'Latest'
+
+& (Join-Path -Path $PSScriptRoot -ChildPath 'Initialize-WhiskeyTest.ps1' -Resolve)
+
+$testRoot = $null
+$output = $null
+
+function Init
+{
+    $Global:Error.Clear()
+    $script:testRoot = New-WhiskeyTestRoot
+}
+
+function GivenModuleInstalledLocally
+{
+    Mock -CommandName 'Test-Path' -ModuleName 'Whiskey' -MockWith { $true }
+    Mock -CommandName 'Import-Module' -ModuleName 'Whiskey' -MockWith { $true }

```

Comment:
Move this `Import-Module` mock to the `Init` function as we should always mock it, we don't need to actually import any modules for the tests.